apps/saleor-app-checkout/package.json

```diff
@@ -53,8 +54,9 @@
     "retes": "^0.30.0",
     "stripe": "10.2.0",
     "ts-invariant": "0.10.3",
-    "url-join": "^5.0.0",
-    "urql": "^2.2.0"
+    "url-join": "4.0.1",
```

Comment:
Why did we downgrade `url-join` from 5.0 to 4.0? 🤔

Reply:
5.0 is esm-module ONLY. It worked with Next.js somewhat by accident but in general, it's problematic.